packages/lib-classifier/src/components/Classifier/components/QuickTalk/QuickTalkContainer.js

```diff
@@ -144,21 +94,21 @@ function QuickTalkContainer ({
 
       if (existingDiscussion) { // Add to the existing discussion
 
-        const response = await postTalkComment(text, existingDiscussion, user, authorization)
-        if (!response?.ok) throw new Error(t('QuickTalk.errors.failPostComment'))
+        const newComment = await postTalkComment(text, existingDiscussion, user, authorization)
+        if (!newComment) throw new Error(t('QuickTalk.errors.failPostComment'))
+        comments.push(newComment)
 
         setPostCommentStatus(asyncStates.success)
         setPostCommentStatusMessage('')
-        fetchComments()
 
       } else {  // Create a new discussion
 
-        const response = await postTalkDiscussion (text, discussionTitle, subject, defaultBoard, user, authorization)
-        if (!response?.ok) throw new Error(t('QuickTalk.errors.failPostDiscussion'))
+        const discussion = await postTalkDiscussion (text, discussionTitle, subject, defaultBoard, user, authorization)
```

Comment:
Some odd whitespace here, when the function is invoked. It’s ignored but looks a bit strange when you read it.